_episodes/01-introduction.md

```diff
@@ -24,6 +24,7 @@ keypoints:
 - "Deep Learning is well suited to classification and prediction problems such as image recognition."
 - "To use Deep Learning effectively we need to go through a workflow of: defining the problem, identifying inputs and outputs, preparing data, choosing the type of network, choosing a loss function, training the model, tuning Hyperparameters, measuring performance before we can classify data."
 - "Keras is a Deep Learning library that is easier to use than many of the alternatives such as TensorFlow and PyTorch."
+math: true
```

Comment:
Does this do anything?

Reply:
I would remove it.

Reply:
it's required if we want to use formatted equation.